callautomation-live-transcription/main.py

```diff
@@ -1,15 +1,14 @@
 
 import ast
```

Comment:
Please test the application and share the correlation id's along with the description

Reply:
connection ID: 0f004480-228e-4727-9d25-b7463bacb305
'correlationId': '0f8504ff-b328-4a50-9f70-3b4e057aa351'